server/db/models/Appointment.js

```diff
@@ -0,0 +1,34 @@
+const mongoose = require("mongoose");
+
+const Schema = mongoose.Schema;
+
+const schema = new Schema({
+    meeting_id: {
+        // TODO: generate unique meeting id
+        // unique: true,
+        type: String,
```

Comment:
This should probably be a Schema.Types.ObjectId so that we can link it directly to the meeting type. Also Mongoose will automatically generate a unique id for us

Reply:
Ah, I thought meeting ids would be readable but unique to the user, like the calendly link with an id of something like '/team-sprint' you pointed out

Reply:
That's an interesting point, let's discuss in the standup